cmp/cmpopts/equate.go

```diff
@@ -123,3 +124,36 @@ func (a timeApproximator) compare(x, y time.Time) bool {
 	// Note: time.Time doesn't have AfterOrEqual method hence the negation.
 	return !x.Add(a.margin).Before(y)
 }
+
+// AnyError is an error that matches all other non-nil errors.
+var AnyError anyError
+
+type anyError struct{}
+
+func (anyError) Error() string     { return "any error" }
+func (anyError) Is(err error) bool { return err != nil }
+
+// EquateErrors returns a Comparer option that determines errors to be equal
+// if errors.Is reports them to be equal. The AnyError error can be used to
```

Comment:
"...if errors.Is reports them to match."

I've been trying (not always successfully) to use the verb "match" for the test performed by errors.Is.

Reply:
Done.